.github/workflows/3x.yml

```diff
@@ -58,7 +58,7 @@ jobs:
     strategy:
       matrix:
         operating-system: [ 'macos-latest' ]
-        php-versions: [ '7.4', '8.2', '8.3' ]
+        php-versions: [ '7.4', '8.3', '8.4' ]
```

Comment:
We still support/recommend PHP 8.2... Should we continue to test against that and just add 8.4 instead?

Reply:
PHP Terminus supports != PHP platform supports.

If we're not going to test all supported versions of PHP, go with lowest and highest (8.1 and 8.4), otherwise matrix all of them.

Reply:
Yeah I don't think that Terminus needs to test against every PHP version, just a representative sample should be fine. I'd recommend 7.4, 8.1 (or whatever the lowest php version that is still supported) and 8.4 (or the highest version of php supported by Terminus).